src/System Application/App/AI/src/Copilot/CopilotAICapabilities.Page.al

```diff
@@ -346,6 +410,21 @@ page 7775 "Copilot AI Capabilities"
         CopilotTelemetry.SendCopilotDataMovementUpdatedTelemetry(AllowDataMovement);
     end;
 
+    local procedure UpdateBingSearchOptIn()
+    var
+        SystemPrivacyNoticeReg: Codeunit "System Privacy Notice Reg.";
+        BingSearchNudgeNotification: Notification;
+    begin
+
+        if BingOptIn then
+            PrivacyNotice.SetApprovalState(SystemPrivacyNoticeReg.GetBingPrivacyNoticeName(), "Privacy Notice Approval State"::Agreed)
+        else begin
+            PrivacyNotice.SetApprovalState(SystemPrivacyNoticeReg.GetBingPrivacyNoticeName(), "Privacy Notice Approval State"::Disagreed);
+            BingSearchNudgeNotification.Message('You''re missing out! Enabling Bing Search offers enhanced results');
```

Comment:
Also add a fixed guid id for the notification. Otherwise this might show up multiple times if they enable and disable more than once.